service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -479,19 +537,37 @@ private static MethodVersionDetails methodVersionToMethodVersionDetails(
         .description(methodVersion.description())
         .created(DateUtils.convertToDate(methodVersion.created()))
         .lastRun(initializeLastRunDetails(methodVersion.lastRunSetId()))
-        .url(methodVersion.url());
+        .url(methodVersion.url())
+        .branchOrTagName(methodVersion.branchOrTagName());
   }
 
-  private MethodDetails methodVersionToMethodDetails(MethodVersion methodVersion) {
+  private MethodDetails methodVersionToMethodDetails(
+      MethodVersion methodVersion, Boolean includeSourceDetails) {
     Method method = methodVersion.method();
+    GithubMethodSourceDetailsResponse sourceDetails;
+
+    if (includeSourceDetails != null) {
```

Comment:
Since `includeSourceDetails` should this essentially be this similar to above?
```
if (includeSourceDetails) ..
```

Reply:
Looking more closely it seems like I get different behavior when I set the variable to `boolean` or to `Boolean`. `boolean` throws ```Operator '!=' cannot be applied to 'boolean', 'null'``` while `Boolean` does not. I think I'll change both instances to `Boolean` to have that extra check.